rebar.lock

```diff
@@ -69,6 +69,7 @@
  {<<"unicode_util_compat">>,{pkg,<<"unicode_util_compat">>,<<"0.7.0">>},2},
  {<<"webdavfilez">>,{pkg,<<"webdavfilez">>,<<"1.2.1">>},0},
  {<<"yamerl">>,{pkg,<<"yamerl">>,<<"0.8.1">>},0},
+ {<<"zotonic_apps">>,{pkg,<<"zotonic_apps">>,<<"1.0.0-rc.10">>},0},
```

Comment:
As this is the master Zotonic repo, do we need this dependency?

Reply:
These get added on `make` on my machine, because I have a project in `apps_user` (atlas) that has `zotonic_apps` as dependency. We probably do not want site dependencies to end up in here. Do you know any way to fix that?

Reply:
I tend to update the zotonic master dependencies in a separate directory. This prevents that development apps are coming along.